geoips/schema/plugin_validator.py

```diff
@@ -73,7 +80,13 @@ def get_validators(schema_dict):
 def validate(plugin_file):
     """Validate the a YAML-based plugin."""
     plugin_yaml = yaml.safe_load(open(plugin_file, "r"))
-    validator = validators[plugin_yaml["interface"]]
+    family = plugin_yaml["metadata"]["family"]
+    validator_name = plugin_yaml["interface"]
+    print(f"Checking validator {validator_name}.{family}")
+    if f"{validator_name}.{family}" in validators:
```

Comment:
I am pulling the interface and family from the plugins themselves though.  Just using the path to get it from the schema.